.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_CLIENT_ID=517724740678-2uvsschghjg4nnqpejrfbpc9db2i8226.apps.googleusercontent.com
```

Comment:
is the client_id sensitive information? if so, maybe we shouldn't push it & add .env to .gitignore

Reply:
It is, but that means everyone who's working on the project would need to manually generate this file. I think a better solution since we're so short on time is to make the repository private for now and add the value to heroku later

Reply:
I think we can copy the client Id ourselves. The other PR adds .env to the gitignore.